packages/machine-extractor/src/utils.ts

```diff
@@ -431,3 +431,13 @@ function hash(str: string): string {
     .toString(32)
     .substring(1, 10);
 }
+
+export const isErrorWithMessage = (
```

Comment:
I'd move this to be local util, close to the server stuff (as it's used there). We could have this in some kind of a shared thing but I don't think that machine-extractor package is that. This util falls outside of its domain and shouldn't be part of its public API